backend/worker/opensearch.go

```diff
@@ -48,25 +49,54 @@ func (w *Worker) IndexSessions(ctx context.Context, isUpdate bool) {
 		return nil
 	}
 
-	whereClause := "True"
+	start := time.Now()
```

Comment:
cool, so this is using the time to make the fetch, processed, update atomic, right?